docs/hub/spaces-sdks-docker-quarto.md

```diff
@@ -0,0 +1,24 @@
+# Quarto on Spaces
+
+Quarto is a powerful tool for creating reproducible scientific and technical documents.
+With Quarto you can interleave markdown and code chunks to generate pdfs, websites, presentations and more.
+
+### Getting Started with Quarto on Spaces
+
+1. Create the Quarto space, and clone it locally
+
+<a  href="https://huggingface.co/new-space?template=posit/quarto-template"> <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg"/> </a>
```

Comment:
I would split this into two steps, and I think you should add an example code snippet on how to clone the repo

Reply:
Actually, cloning locally is not a strict requirement no? People can do the development entirely in the UI. I would suggest to split this into

* Creating and setting up the repo, doing changes in the Space by directly modifying `/src`
* Developing locally (install quarto, VS Code extension)
* Learn more about  Quarto

Reply:
I suppose that's true, but working on a quarto site in the browser without being able to preview the site. Happy to change the organization though.

Reply:
I think mentioning, even if briefly, the in-browser setup could be ok!